specification/batch/resource-manager/Microsoft.Batch/stable/2020-03-01/BatchManagement.json

```diff
@@ -1731,11 +1731,22 @@
             "description": "The operation was successful. The response contains the PrivateEndpointConnection.",
             "schema": {
               "$ref": "#/definitions/PrivateEndpointConnection"
-            },
+            }
+          },
+          "204": {
+            "description": "The operation was successful."
+          },
+          "202": {
```

Comment:
making a sync call as async is a breaking change and requires a new api-version

Reply:
To be clear here @ravbhatnagar the underlying API did not change. The previous swagger did not match the FE behavior for the API, which is why these changes are being made.